nav2_core/include/nav2_core/recovery.hpp

```diff
@@ -74,6 +74,9 @@ class Recovery
   // TODO(stevemacenski) evaluate design for recoveries to not host
   // their own servers and utilize a recovery server exposed action.
   // virtual bool executeRecovery() = 0;
+
+protected:
+  double transform_tolerance_;
```

Comment:
Sorry, you misintrepseted me, I meant the recovery.hpp file in the nav2_recoveries package that implements the backend recovery server. That's where this variable should live and where the get-parameter should happen

Reply:
Ops my bad, should be fixed now.